docs/source/use_cases/discover_public_data/clinvar_annotations.rst

```diff
@@ -0,0 +1,18 @@
+ClinVar Annotations
+===================
+
+Annotations from `ClinVar`_ were loaded into Google Genomics for use in sample annotation pipelines.  This data reflects the state of `ClinVar`_ at a particular point in time - it is not being kept current as more annotations are added to `ClinVar`_ over time.
```

Comment:
I don't think that the " - it is not being kept current..." is necessary.  The first part of the sentence is clear.
On the other hand, if you expect to have a policy for updating on some periodic basis, it might be worth mentioning that.